packages/jest-haste-map/src/index.js

```diff
@@ -39,6 +39,7 @@ type Options = {
   cacheDirectory?: string,
   console?: Console,
   extensions: Array<string>,
+  forceNativeFind?: boolean,
```

Comment:
Agreed - `native` is a bad choice of wording here, but `NodeFSApi` is somewhat misleading because this option forces the crawler to use something that is _specifically_ avoiding the Node FS API. Spawning a process that calls a system `find` command and reading its standard output as opposed to calling one of the `read` methods on the `fs` module (`readdir` in this case) is patently the opposite of what "`forceNodeFSApi`" would suggest. How do you feel about `forceSystemFind`?